.github/workflows/deploy.yml

```
@@ -52,6 +52,7 @@ jobs:
             --no-progress
             --base "$(pwd)/build"
             --accept 100..399,401,403,429
+            --exclude '^.*/status/migration/.*$'
```

Should we still have the domain in the URL?

```suggestion
            --exclude 'https://conda-forge.org/status/migration/.*$'
```